### DISCUSSION TITLE
Calls to certain table suddenly not working in production

### DISCUSSION THREAD

**Discussion by NathanVG** [planetscale/discussion#549] [Bug Reports]
I have a next.js app using the T3 stack with TRPC and Prisma, my db is hosted on planetscale.

Since yesterday calls to a specific table are suddenly not resolving. Locally everything works perfectly and every call goes through. However for my production deploy on Vercel these calls are simply not doing anything, they keep loading. I'm not getting any errors.

Everything works fine locally, my data seems to be in order through prisma studio as well. I updated all packages yesterday and just now again as I saw a new release for the Prisma client was available.

I accessed previous deploys from when everything was working, and the issue persists, so either it is a planetscale issue or a vercel issue

I'm at wits end, anyone that can make a suggestion?

more info:
https://github.com/prisma/prisma/issues/21368

**Comment by mscoutermarsh** [comment]
Hi @NathanVG,

One suggestion. Are you able to see what the SQL query being sent to us is? One way to narrow down where the problem is: you could grab that query and then try running it directly in the PlanetScale web console & see what the results are.

**Comment by NathanVG** [comment]
Hi @mscoutermarsh, 

this issue seemingly resolved itself after about 24 hours. My guess is it was a hanging Vercel lambda function  that got restarted. 
I still have no idea why this happened or how it was resolved. 

For posterity here are the links to the issues I started in the Vercel and Prisma repo's:
https://github.com/prisma/prisma/issues/21368#issuecomment-1749648684
https://github.com/orgs/vercel/discussions/4368#discussioncomment-7196111
